lib/hyrax/iiif_av/spec/shared_specs/displays_content.rb

```diff
@@ -71,6 +72,7 @@
           expect(content).to all(be_instance_of IIIFManifest::V3::DisplayContent)
           expect(content.length).to eq 2
           expect(content.map(&:type)).to all(eq 'Video')
+          expect(content.map(&:type)).to all(eq 'video/mp4')
```

Comment:
Looks like a copy/paste issue.
```suggestion
          expect(content.map(&:format)).to all(eq 'video/mp4')
```